tests/data/claypit/c3.php

```diff
@@ -229,18 +243,37 @@ function __c3_factory($filename, $lock=false)
             } else {
                 $phpCoverage = unserialize(file_get_contents($filename));
             }
-            
-            return array($phpCoverage, $file);
-        } else {
-            if (method_exists(Driver::class, 'forLineCoverage')) {
-                //php-code-coverage 9+
+
+            return array(
```

Comment:
please, let's follow the boy scout rule here as much as possible.
Stop using `array( )`, use `[ ]` instead.

Reply:
This file is copy of https://github.com/Codeception/c3/blob/main/c3.php
which must stay compatible with PHP 5.5 and Codeception 3.

Please review modernization PR at https://github.com/Codeception/c3/pull/72

Reply:
well, he's asking if he should remove the compatibility logic as well, you know my opinion on that.
Personally, I never used C3 which is why I refrain from commenting.

Reply:
The file in claypit will be updated when we have an updated file in c3 main branch.